src/test/kotlin/racingcar/car/NameTest.kt

```diff
@@ -0,0 +1,29 @@
+package racingcar.car
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class NameTest {
+
+    @Test
+    fun `should fail if car name is longer than five characters`() {
+        Assertions.assertThatThrownBy { Name("Haaallo") }
+            .isInstanceOf(IllegalArgumentException::class.java)
+    }
+
+    @Test
+    fun `should fail if car name is empty`() {
+        Assertions.assertThatThrownBy { Name("") }
+            .isInstanceOf(IllegalArgumentException::class.java)
+    }
```

Comment:
```suggestion
    @Test
    fun `should fail if car name is empty`() {
        assertThrows<IllegalArgumentException> {
            Name("")
        }
    }
```

Maybe this one is more like a Kotlin style?

Reply:
Okay maybe next time..